en-US/firefox/whatsnew_67.0.5.lang

```diff
@@ -0,0 +1,93 @@
+## NOTE: Demo page TBD
+
```

Comment:
Need double line space between entries. Check the entire doc for single-line spaces.

Reply:
Ok, spacing should be fixed now.